src/Identity/Controllers/AccountController.cs

```diff
@@ -1,62 +1,75 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Security.Claims;
-using System.Threading.Tasks;
-using Bit.Core.Models.Table;
+﻿using Bit.Core.Models.Table;
 using Bit.Core.Repositories;
+using Bit.Core.Services;
 using Bit.Identity.Models;
 using IdentityModel;
 using IdentityServer4;
 using IdentityServer4.Services;
 using IdentityServer4.Stores;
 using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
 
 namespace Bit.Identity.Controllers
 {
     public class AccountController : Controller
     {
-        private readonly IIdentityServerInteractionService _interaction;
-        private readonly IUserRepository _userRepository;
-        private readonly ISsoConfigRepository _ssoConfigRepository;
         private readonly IClientStore _clientStore;
+        private readonly IIdentityServerInteractionService _interaction;
         private readonly ILogger<AccountController> _logger;
+        private readonly ISsoConfigRepository _ssoConfigRepository;
+        private readonly IUserRepository _userRepository;
 
         public AccountController(
+            IClientStore clientStore,
             IIdentityServerInteractionService interaction,
-            IUserRepository userRepository,
+            ILogger<AccountController> logger,
+            IOrganizationUserRepository organizationUserRepository,
             ISsoConfigRepository ssoConfigRepository,
-            IClientStore clientStore,
-            ILogger<AccountController> logger)
+            IUserRepository userRepository,
+            IUserService userService)
         {
-            _interaction = interaction;
-            _userRepository = userRepository;
-            _ssoConfigRepository = ssoConfigRepository;
             _clientStore = clientStore;
+            _interaction = interaction;
             _logger = logger;
+            _ssoConfigRepository = ssoConfigRepository;
+            _userRepository = userRepository;
         }
 
         [HttpGet]
         public async Task<IActionResult> Login(string returnUrl)
         {
             var context = await _interaction.GetAuthorizationContextAsync(returnUrl);
-            if (context.Parameters.AllKeys.Contains("domain_hint") &&
-                !string.IsNullOrWhiteSpace(context.Parameters["domain_hint"]))
-            {
-                return RedirectToAction(nameof(ExternalChallenge),
-                    new { organizationIdentifier = context.Parameters["domain_hint"], returnUrl = returnUrl });
-            }
-            else
+
+            var domainHint = context.Parameters.AllKeys.Contains("domain_hint") ? context.Parameters["domain_hint"]
```

Comment:
Sorry to nit.. Format should be:
```C#
var domainHint = context.Parameters.AllKeys.Contains("domain_hint") ?
    context.Parameters["domain_hint"] : null;
```

Reply:
All good. I've always seen it the way from before:
```
bool
    ? x
    : y
```
So seeing it like this was strange, but the mdn docs backed you up (for javascript at least, everything I saw on MS was done on one line)... I still think it looks weird 😂 but must concede. 

Fixed.